### DISCUSSION TITLE
Discussion on project.

### DISCUSSION THREAD

**Discussion by saswatsamal** [hackclubiter/onEditor#2] [General]
# About
From today we will start our discussion, research and documentation for this project.
Google Meet Link : [Click Here](https://meet.google.com/wxx-iane-tgc)

# Discussion Topics
- How to start?
- 1 month for documentation and research
- Where to start?
- References

**Comment by saswatsamal** [comment]
It'll start at 8PM IST on 31st March, 2021

**Comment by Saswat7101** [comment]
Ok

**Comment by saswatsamal** [comment]
I want resources to start with. If anyone can list down the things we will have to learn then it'll be great.
CC: @RahulAnand442001 @LoopGlitch26

**Reply by Saswat7101** [reply]
Yes if we can list down things that we have to learn and in a different discussion we can add all the resources related to a particular subj or tech

**Comment by saswatsamal** [comment]
I know many of you have used the *programiz.com* and they have got an outstanding ide. I hope [this](https://www.programiz.com/blog/online-python-compiler-engineering/) blog from them will help in getting started with it.

**Comment by LoopGlitch26** [comment]
As a headstart let's try to integrate [JudgeO widget](https://github.com/judge0/widgets) in a simple IDE that we make.

**Reply by LoopGlitch26** [reply]
Keep threading your views and ideas. 

As @Saswat7101 had suggested, I also believe that we can start by making an IDE with an existing API or integrated system **first**. It will give us a broad idea about how to make a successful IDE. After that we can go for the Compiler which will require heavy backend

**Comment by ankitrajbiswal** [comment]
There are lot of you tube videos which are related to our project, I saw couple of videos, should I post those links here???

**Reply by LoopGlitch26** [reply]
Yes, sure! If you have too many resources, then you can start a new discussion mentioning "YouTube Videos on Online Compilers" or something like that. Or you can list them here, as you wish.
